_security/audit-logs/storage-types.md

```diff
@@ -108,6 +108,16 @@ Name | Data type | Description
 `plugins.security.audit.config.webhook.format` | String | The format in which the audit log message is logged, can be one of `URL_PARAMETER_GET`, `URL_PARAMETER_POST`, `TEXT`, `JSON`, `SLACK`. See [Formats](#formats).
 
 
+### Basic auth settings
```

Comment:
The contents of this PR are good, but I think we can make these inline with the table above.